Neos.Flow/Classes/Http/Client/CurlEngine.php

```diff
@@ -107,6 +107,14 @@ public function sendRequest(Http\Request $request)
                     ]);
                 }
             break;
+            case 'HEAD':
+                if (!is_resource($content)) {
+                    $body = $content !== '' ? $content : http_build_query($request->getArguments());
+                    curl_setopt($curlHandle, CURLOPT_POSTFIELDS, $body);
+                }
+                curl_setopt($curlHandle, CURLOPT_CUSTOMREQUEST, $request->getMethod());
```

Comment:
Not needed, since `CURLOPT_NOBODY` sets the method to `HEAD` automatically, see https://secure.php.net/manual/en/function.curl-setopt.php:
> CURLOPT_NOBODY: TRUE to exclude the body from the output. Request method is then set to HEAD.